File: Front/Models/UserAccount.cs
namespace Front.Models
{
    public class UserAccount
    {
        public int AccountNumber { get; set; }
        public string Alias { get; set; }
        public string AccountType { get; set; }
        public string Currency { get; set; }
        public decimal Balance { get; set; }
        public string? CBU { get; set; } //In case of CryptoCurrency, is null.
        public string? UUID { get; set; } //In case of fiduciary, is null.
    }
}

File: Front/wwwroot/js/UserAccount.js
var token = window.token;

let table = new DataTable('#userAccounts', {

    ajax: {
        url: "https://localhost:7295/api/Account/GetUserAccounts",
        dataSrc = "data"
        headers: {"Authorization": "Bearer " + token}
    },
    columns: [
        { data: 'accountNumber', title: 'AccountNumber' },
        { data: 'alias', title: 'Alias' },
        { data: 'accountType', title: 'AccountType' },
        { data: 'currency', title: 'Currency' },
        { data: 'balance', title: 'Balance' },
        { data: 'cbu', title: 'CBU' },
        { data: 'uuid', title: 'UUID'},
    ]
})